extensions/adobe/experience/adcloud/cmpeventattribution.schema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/cmpeventattribution",
+  "title": "Advertising Cmp Event Attribution Details",
+  "type": "object",
+  "description": "Advertising Cmp Event Attribution details such as dsp buy, dsp site, dsp page.",
+  "definitions": {
+    "cmpeventattribution": {
+      "properties": {
+        "xdm:experienceId": {
+          "title": "Experience Identifier",
+          "type": "number",
```

Comment:
Should this be a string instead?

Reply:
same for the other IDs - are they always numbers?  And if numbers, are they really integers?  And should there be min/max?

Reply:
I changed all ids to strings for more flexibility.